msteams-platform/apps-in-teams-meetings/API-references.md

```diff
@@ -372,29 +380,108 @@ GET /v1/meetings/{meetingId}
+                        "joinUrl": "https://teams.microsoft.com/l/xx", 
+                        "title": "All Hands", 
+                        "type": "Scheduled" 
+                    }, 
+        "conversation": { 
+                            "isGroup": true, 
+                            "conversationType": "groupchat", 
+                            "id": "meeting chat ID" 
+                        }, 
+        "organizer": { 
+                        "id": "<organizer user ID>", 
+                        "aadObjectId": "<AAD object ID>",
+                        "objectId": "<organizer object ID>",
+                        "tenantId": "<Tenant ID>" 
+                    }
+    } 
+    ```
+
+* **One-on-one calls:**
+
+    ```json
+    {
+        "details": {
+                        "id": "<meeting ID>",
+                        "type": "OneToOneCall"
+                    },
+        "conversation": {
+                            "isGroup": true,
+                            "conversationType": "groupChat",
+                            "id": "meeting chat ID"
+                        },
+        "organizer  ": {
+                            "id": "<organizer user ID>",
+                            "aadObjectId": "<AAD object ID>",
+                            "objectId": "<organizer object ID>",
+                            "tenantId": "<Tenant ID>" 
+                        }
     }
-} 
-```
+    
+    ```
+
+* **Group calls:**
+
+    ```json
+    {
+        "details": {
+                        "id": "<meeting ID>",
+                        "type": "GroupCall",
+                        "joinUrl": "https://teams.microsoft.com/l/xx"
+                    },
+        "conversation": {
+                            "isGroup": true,
+                            "conversationType": "groupChat",
+                            "id": "meeting chat ID"
+                        },
+        "organizer": {
+                        "id": "<organizer user ID>",
+                        "objectId": "<organizer object ID>",
+                        "aadObjectId": "<AAD object ID>",
+                        "tenantId": "<Tenant ID>" 
+                     }
+    }
+    
+    ```
+
+* **Instant meetings:**
+
+    ```json
+    { 
+       "details": { 
+                        "id": "<meeting ID>", 
+                        "msGraphResourceId": "MSowYmQ0M2I4OS1lN2QxLTQxNzAtOGZhYi00OWJjYjkwOTk1YWYqMCoqMTk6bWVldGluZ19OVEkyT0RjM01qUXROV1UyW", 
+                        "scheduledStartTime": "2022-04-24T22:00:00Z", 
+                        "scheduledEndTime": "2022-04-24T23:00:00Z", 
+                        "joinUrl": "https://teams.microsoft.com/l/xx", 
+                        "title": "All Hands", 
+                        "type": "MeetNow" 
+                    }, 
+        "conversation": { 
+                            "isGroup": true, 
+                            "conversationType": "groupchat", 
```

Comment:
Does the casing of this value really differ across the meeting types? In the examples we have both `groupchat` and `groupChat`.